src/main/java/pl/betoncraft/betonquest/conversation/ConversationData.java

```diff
@@ -17,9 +17,10 @@
  */
 package pl.betoncraft.betonquest.conversation;
 
-import java.util.ArrayList;
-import java.util.HashMap;
+import java.util.*;
```

Comment:
I do prefer single imports so that you can directly see the full class name of all used imports when looking at the source on github. 

I know it's a small thing and grouping inports is IntelliJs default setting but I would love if you could change adapt [these code style settings](https://user-images.githubusercontent.com/17405009/46570707-ddd05580-c968-11e8-9bf4-6dc1b5a9e849.png) for the future...